pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta:__legacy__"
```

Comment:
I don't know why this file is required. Did you add it on purpose?

Reply:
No, this file just appeared, when i installed xhtml2pdf. I think it is kind of configuration file for packaging.